swagger.yaml

```diff
@@ -1732,4 +1918,103 @@ components:
         syncNumber:
           type: integer
           format: int32
-          description: Sync number
\ No newline at end of file
+          description: Sync number
+
+    ProductInspectionTemplate:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+          description: Product inspection template id in UUID format
+        isActive:
+          type: boolean
+          description: Is product inspection template active
+        inspectionInterval:
+          type: number
+          format: float
+          description: Product inspection template object
+        fields:
+          type: array
+          items:
+            $ref: "#/components/schemas/ProductInspectionTemplateField"
+          description: List of product inspection template fields
+        metadata:
+          $ref: "#/components/schemas/Metadata"
+
+    ProductInspectionTemplateField:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+          description: Product inspection template field id in UUID format
+        unit:
+          type: string
+          description: Product inspection template field unit
+        minTolerance:
+          type: number
+          format: float
+          description: Product inspection template field min tolerance
+        maxTolerance:
+          type: number
+          format: float
+          description: Product inspection template field max tolerance
+        measurementNumber:
+          type: integer
+          description: Product inspection template field measurement number
+        nominalNumber:
+          type: number
+          format: float
+          description: Product inspection template field nominal number
+
+    ProductInspection:
+      type: object
+      description: Product inspection object
+      properties:
+        fields:
+          type: array
+          items:
+            $ref: "#/components/schemas/ProductInspectionField"
+          description: List of product inspection fields
+        externalLink:
+          type: string
+          description: Product inspection external link
```

Comment:
What is this used for?

Reply:
Lisäoptiona linkki esim. google-sheetsiin tms. erillisessä osoitteessa olevaan tarkastuslomakkeeseen, johon syötetään tarkastuksen tulokset. 
I think it's supposed to be a link to some other document that has the results filled in?